modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_data.rb

```diff
@@ -99,27 +99,27 @@ def rep_last_name
         def rep_phone_area_code
           return if rep_country_code != '1'
 
-          higher_level_review.rep_phone_data&.dig('areaCode') || ''
+          higher_level_review.informal_conference_rep_phone.area_code || ''
         end
 
         def rep_phone_prefix
           return if rep_country_code != '1'
 
-          higher_level_review.rep_phone_data&.dig('phoneNumber')&.first(3) || ''
+          higher_level_review.informal_conference_rep_phone.phone_number&.first(3) || ''
```

Comment:
What do you think about pushing this phone number logic (for several of these related methods) down at least one layer?

Reply:
down a layer, into the `Phone` class, you mean?

Reply:
Yeah, that's what I was thinking. Instead of `&.first(3)` it could be `area_code`, `prefix`, etc... I suppose the next question is - who's responsibility is it to return the empty string? The phone class or the `FormData#rep_phone_prefix` method, I'm not sure what I think about that...